src/components/app/DocumentList/index.js

```diff
@@ -0,0 +1,855 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { push } from 'react-router-redux';
+import { Map, Set } from 'immutable';
```

Comment:
iMap

Reply:
iSet

Reply:
I don't get this comment. What `iMap` ? That's some different API, or just a different name ?